modules/installation-custom-aws-vpc.adoc

```diff
@@ -60,16 +69,43 @@ Record each subnet ID. Completing the installation requires that you enter these
 ** The public subnet requires a NAT gateway with an EIP address.
 ** The private subnet requires a route to the NAT gateway in public subnet.
 endif::public[]
+ifdef::aws-outposts[]
+[NOTE]
+====
+To allow the creation of {product-title} with remote workers in the AWS Outposts, you must create at least one private subnet in the AWS Outpost instance for the workload instances creation and one private subnet in an AWS region for the control plane instances creation. If you specify more than one private subnet in the region, the control plane instances will be distributed across these subnets. You will also need to create a public subnet in each of the availability zones used for private subnets, including the Outpost private subnet, as Network Load Balancers will be created in the AWS region for the API server and Ingress network as part of the cluster installation. It is possible to create an AWS region private subnet in the same Availability zone as an Outpost private subnet.
+====
+* Create a public and private subnet in the AWS Region for each availability zone that your control plane uses. Each availability zone can contain no more than one public and one private subnet in the AWS region. For an example of this type of configuration, see link:https://docs.aws.amazon.com/vpc/latest/userguide/VPC_Scenario2.html[VPC with public and private subnets (NAT)] in the AWS documentation.
++
+To create a private subnet in the AWS Outposts, you need to first ensure that the Outpost instance is located in the desired availability zone. Then, you can create the private subnet within that availability zone within the Outpost instance, by adding the Outpost ARN. Make sure there is another public subnet in the AWS Region created in the same availability zone.
++
+Record each subnet ID. Completing the installation requires that you enter all the subnets IDs, created in the AWS Region, in the `platform` section of the `install-config.yaml` file and changing the workes machineSet to use the private subnet ID created in the Outpost. See link:https://docs.aws.amazon.com/managedservices/latest/userguide/find-subnet.html[Finding a subnet ID] in the AWS documentation.
++
+[IMPORTANT]
+====
+In case you need to create public subnet in the AWS Outposts, you need to make sure this subnet is not going to be used for the Network/Classic LoadBalancer, otherwise the LoadBalancer creation would fail. To achieve that, the `kubernetes.io/cluster/.*-outposts: owned` special tag must be included in the subnet.
```

Comment:
```suggestion
In case you need to create public subnet in the AWS Outposts, verify that this subnet is not used for the Network or Classic LoadBalancer, otherwise the LoadBalancer creation fails. To achieve that, the `kubernetes.io/cluster/.*-outposts: owned` special tag must be included in the subnet.
```

Reply:
A few minimalism suggestions

Reply:
Thanks @skrthomas I've applied your feedback.